src/containers/IntentService.jsx

```diff
@@ -31,6 +32,12 @@ export default class IntentService extends Component {
           throw new Error('Unexpected data from intent')
         }
 
+        if (data && data.timeout === true) {
```

Comment:
I would suggest to use `data.successTimeout` again here because we can think about a limit to run the job itself and it's not the case here.